### PR TITLE
Remove the `hex` field from `listpresignedtxs`

### PR DESCRIPTION
**Pull request by darosior**
It was redundant, and needlessly added complexity and computation. 

Based on #367. Only the last commit is part of this PR.

### CODE REVIEW

**Review by edouardparis — APPROVED** [review]
ACK 6570c5b97abebd678b85e84d6ce7772840e2fdb7

**Comment by darosior** [comment]
Arg i overlooked that it means we need a Miniscript finalizer for the Cancel PSBT in the functional tests..

**Comment by darosior** [comment]
Finally back to this. I use the [Miniscript satisfier i implemented in Python](https://github.com/darosior/python-bip380) to finally get rid of the `"hex"` field.

File: doc/API.md [hunk 1]
```diff
@@ -188,13 +188,13 @@ vault's state).
...
-| `unvault_emergency` | [presigned_tx](#presigned-tx)                                  | The Unvault Emergency transaction, or `null` if we are not a stakeholder |
+| Field               | Type     | Description                                                                                    |
+| ------------------- | -------- | ---------------------------------------------------------------------------------------------- |
+| `vault_outpoint`    | string   | The vault deposit transaction outpoint.                                                        |
+| `unvault`           | string   | The Unvaulting transaction PSBT (base64 encoded)                                               |
+| `cancel`            | string   | The Cancel transaction PSBT (base64 encoded)                                                   |
+| `emergency`         | string   | The Emergency transaction PSBT (base64 encoded), or `null` if we are not a stakeholder         |
+| `unvault_emergency` | string   | The Unvault Emergency transaction PSBT (base64 encoded), or `null` if we are not a stakeholder |
```
Comment (darosior) [inline doc/API.md:197]: Forgot to remove the "presigned txs" section below

Comment (darosior) [reply doc/API.md:197]: Done

**Review by edouardparis — APPROVED** [review]
ACK b64ccdb5e2ee7c85849bb5ef84883c5eec3e1564

**Pull request merged**